packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -157,8 +161,11 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
             return;
         }
 
+        // if the popover handle exists, take it into account
+        const popoverHandleAdjustmentFactor = this.state.isTruncated ? this.popoverComponent.state.targetWidth : 0;
```

Comment:
Yeah I wasn't sure what else to do here. The other option was to modify popover so I could pass a callback that would pass up the ref (what react more or less says to do in their docs) but I didn't want to modify the popover in a table PR if at all possible. I can do that instead though.